### DISCUSSION TITLE
Tidepool

### DISCUSSION THREAD

**Discussion by hanbozdugan** [NightscoutFoundation/xDrip#3181] [General]
Attention xDrip+ Developers!!! This is a response from tidepool to my query, why basalt insulin is not uploaded in the basalt column, but all insulins are uploaded in the bolus column:                                                                                                                                                         
![Screenshot 1](https://github.com/NightscoutFoundation/xDrip/assets/8653097/3e7b0568-094f-4ade-8918-59b756be2983)
I  have reviewed your upload history in Tidepool and I can see you are  using either AndroidAPS or xDrip+. You must ask the developers of the  app to change how they send this data to Tidepool because the insulin  delivery values are being sent to us as a bolus not basal.

**Comment by jamorham** [comment]
Are these manually entered basal injections?

**Reply by hanbozdugan** [reply]
I hook them up with Novopen Echo Plus with NFC communication
